solidity/scripts/liquidity-rewards-starter.js

```diff
@@ -39,105 +41,72 @@ const mintAndApproveLPReward = async (
-
+    const lpRewardsTBTCSaddle = await LPRewardsTBTCSaddle.deployed()
     const reward = web3.utils.toWei("1000000")
 
-    await initLPRewardContract(
-      lpRewardsKEEPETH,
-      keepToken,
-      lpRewardsOwner,
-      reward
-    )
-    await initLPRewardContract(
-      lpRewardsTBTCETH,
-      keepToken,
-      lpRewardsOwner,
-      reward
-    )
-    await initLPRewardContract(
-      lpRewardsKEEPTBTC,
-      keepToken,
-      lpRewardsOwner,
-      reward
-    )
-
-    const staker1 = accounts[8]
-    const staker1WrappedTokenBalance = web3.utils.toWei("300")
-
-    const staker2 = accounts[9]
-    const staker2WrappedTokenBalance = web3.utils.toWei("700")
-
-    await mintAndApproveLPReward(
-      KEEPETHWrappedTokenContract,
-      lpRewardsKEEPETH.address,
-      staker1,
-      staker1WrappedTokenBalance
-    )
-    await mintAndApproveLPReward(
-      KEEPETHWrappedTokenContract,
-      lpRewardsKEEPETH.address,
-      staker2,
-      staker2WrappedTokenBalance
-    )
-
-    await mintAndApproveLPReward(
-      TBTCETHHWrappedTokenContract,
-      lpRewardsTBTCETH.address,
-      staker1,
-      staker1WrappedTokenBalance
-    )
-    await mintAndApproveLPReward(
-      TBTCETHHWrappedTokenContract,
-      lpRewardsTBTCETH.address,
-      staker2,
-      staker2WrappedTokenBalance
-    )
-
-    await mintAndApproveLPReward(
-      KEEPTBTCHWrappedTokenContract,
-      lpRewardsKEEPTBTC.address,
-      staker1,
-      staker1WrappedTokenBalance
-    )
-    await mintAndApproveLPReward(
-      KEEPTBTCHWrappedTokenContract,
-      lpRewardsKEEPTBTC.address,
-      staker2,
-      staker2WrappedTokenBalance
-    )
-
-    await lpRewardsKEEPETH.stake(staker1WrappedTokenBalance, { from: staker1 })
-    await lpRewardsKEEPETH.stake(staker2WrappedTokenBalance, { from: staker2 })
-
-    await lpRewardsTBTCETH.stake(staker1WrappedTokenBalance, { from: staker1 })
-    await lpRewardsTBTCETH.stake(staker2WrappedTokenBalance, { from: staker2 })
-
-    await lpRewardsKEEPTBTC.stake(staker1WrappedTokenBalance, { from: staker1 })
-    await lpRewardsKEEPTBTC.stake(staker2WrappedTokenBalance, { from: staker2 })
+    const LP_REWARDS = {
+      KEEP_ETH: {
+        contract: lpRewardsKEEPETH,
+        wrappedTokenContract: null,
+      },
+      TBTC_ETH: {
+        contract: lpRewardsTBTCETH,
+        wrappedTokenContract: null,
+      },
+      KEEP_TBTC: {
+        contract: lpRewardsKEEPTBTC,
+        wrappedTokenContract: null,
+      },
+      TBTC_SADDLE: {
+        contract: lpRewardsTBTCSaddle,
+        wrappedTokenContract: null,
+      },
+    }
+
+    for (const key of Object.keys(LP_REWARDS)) {
+      LP_REWARDS[key].wrappedTokenContract = await getWrappedTokenContract(
+        LP_REWARDS[key].contract
+      )
+
+      await initLPRewardContract(
+        LP_REWARDS[key].contract,
+        keepToken,
+        lpRewardsOwner,
+        reward
+      )
+    }
+
+    for (let i = 8; i < 10; i++) {
```

Comment:
Could we include this loop in the one above so we won't need to do `for (const values of Object.values(LP_REWARDS))` again?